cns/Dockerfile

```diff
@@ -32,8 +32,6 @@ EXPOSE 10090
 
 # skopeo inspect docker://mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0 --format "{{.Name}}@{{.Digest}}"
 FROM mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image@sha256:b4c9637e032f667c52d1eccfa31ad8c63f1b035e8639f3f48a510536bf34032b as windows
-COPY --from=builder /azure-container-networking/cns/kubeconfigtemplate.yaml kubeconfigtemplate.yaml
-COPY --from=builder /azure-container-networking/npm/examples/windows/setkubeconfigpath.ps1 setkubeconfigpath.ps1
```

Comment:
we may want to leave these around in the dockerfile for a bit in case the release of cns which has this removed gets ahead of the overlaymgr release that will remove them from the manifest (or overlaymgr rolls back)

Reply:
I think we won't be releasing CNS urgently, the production impact can be mitigated purely through a daemonset change in AKS

Reply:
Is there a need to keep `cns/kubeconfigtemplate.yaml` and `npm/examples/windows/setkubeconfigpath.ps1` in the repo any longer?

Reply:
likely no but they are referenced in the NPM build, and we need to make sure nothing we're building out of this branch can land on k8s <1.28